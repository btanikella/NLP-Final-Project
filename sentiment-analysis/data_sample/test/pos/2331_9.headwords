adventure
put
Second
members
more
get
