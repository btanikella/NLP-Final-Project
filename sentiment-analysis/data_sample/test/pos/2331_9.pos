RB JJ CC JJ NN WDT VBZ NNP NNP -LRB- RB IN PRP$ JJ JJ NN IN DT NN -RRB- IN DT JJ NNP NN NN WP VBZ TO VB DT JJ NN IN PRP$ JJ  RB : PRP VBZ VBP PRP IN VBG IN DT NNP NNP IN DT NN IN NNS IN DT NN IN JJ  JJ  CC JJ NNS -LRB- CC DT NN  RB -RRB-
NNP NNP NNP NNP -LRB-  NNP NNP  -RRB- CC PRP$ NNS VBP VBN RP DT RB JJ CC JJ NN WDT VBZ TO VB DT NN JJ TO VB  WDT PRP VBP RB RB
RB  IN DT NNP NN  JJS IN DT NN VBP DT CC NN IN DT NN POS NN  WDT VBZ VBN IN DT JJ NN
CD NNS IN DT VBG NN : NNP NNP  NNP NNP  CC NNP NNP VBP DT JJS NNS  WDT VBZ JJ NNS  NNS  CC  RB  PRP$ NNS
NN NN  VBZ JJR IN DT NNP NN IN  NNP NNP  CC  DT NNP NNP
WP DT NN VBZ VBZ VB DT NN VBN DT JJ NN CC PRP VBD PRP
