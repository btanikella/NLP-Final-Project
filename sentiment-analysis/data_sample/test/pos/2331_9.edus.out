Sentence #1 (73 tokens):
Surprisingly good and quick-witted adventure that features Kelsey Grammer (supposedly in his first lead role in a feature) as an unconventional Navy sub commander who wants to run a nuclear sub of his own, however; he has earn it by competing against the U.S. Navy in a series of war-games with a crew of untested, questionable, and lovable men (and a woman, too).
[Text=Surprisingly CharacterOffsetBegin=4 CharacterOffsetEnd=16 PartOfSpeech=RB] [Text=good CharacterOffsetBegin=17 CharacterOffsetEnd=21 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=22 CharacterOffsetEnd=25 PartOfSpeech=CC] [Text=quick-witted CharacterOffsetBegin=26 CharacterOffsetEnd=38 PartOfSpeech=JJ] [Text=adventure CharacterOffsetBegin=39 CharacterOffsetEnd=48 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=49 CharacterOffsetEnd=53 PartOfSpeech=WDT] [Text=features CharacterOffsetBegin=54 CharacterOffsetEnd=62 PartOfSpeech=VBZ] [Text=Kelsey CharacterOffsetBegin=63 CharacterOffsetEnd=69 PartOfSpeech=NNP] [Text=Grammer CharacterOffsetBegin=70 CharacterOffsetEnd=77 PartOfSpeech=NNP] [Text=-LRB- CharacterOffsetBegin=78 CharacterOffsetEnd=79 PartOfSpeech=-LRB-] [Text=supposedly CharacterOffsetBegin=79 CharacterOffsetEnd=89 PartOfSpeech=RB] [Text=in CharacterOffsetBegin=90 CharacterOffsetEnd=92 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=93 CharacterOffsetEnd=96 PartOfSpeech=PRP$] [Text=first CharacterOffsetBegin=97 CharacterOffsetEnd=102 PartOfSpeech=JJ] [Text=lead CharacterOffsetBegin=103 CharacterOffsetEnd=107 PartOfSpeech=JJ] [Text=role CharacterOffsetBegin=108 CharacterOffsetEnd=112 PartOfSpeech=NN] [Text=in CharacterOffsetBegin=113 CharacterOffsetEnd=115 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=116 CharacterOffsetEnd=117 PartOfSpeech=DT] [Text=feature CharacterOffsetBegin=118 CharacterOffsetEnd=125 PartOfSpeech=NN] [Text=-RRB- CharacterOffsetBegin=125 CharacterOffsetEnd=126 PartOfSpeech=-RRB-] [Text=as CharacterOffsetBegin=127 CharacterOffsetEnd=129 PartOfSpeech=IN] [Text=an CharacterOffsetBegin=130 CharacterOffsetEnd=132 PartOfSpeech=DT] [Text=unconventional CharacterOffsetBegin=133 CharacterOffsetEnd=147 PartOfSpeech=JJ] [Text=Navy CharacterOffsetBegin=148 CharacterOffsetEnd=152 PartOfSpeech=NNP] [Text=sub CharacterOffsetBegin=153 CharacterOffsetEnd=156 PartOfSpeech=NN] [Text=commander CharacterOffsetBegin=157 CharacterOffsetEnd=166 PartOfSpeech=NN] [Text=who CharacterOffsetBegin=167 CharacterOffsetEnd=170 PartOfSpeech=WP] [Text=wants CharacterOffsetBegin=171 CharacterOffsetEnd=176 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=177 CharacterOffsetEnd=179 PartOfSpeech=TO] [Text=run CharacterOffsetBegin=180 CharacterOffsetEnd=183 PartOfSpeech=VB] [Text=a CharacterOffsetBegin=184 CharacterOffsetEnd=185 PartOfSpeech=DT] [Text=nuclear CharacterOffsetBegin=186 CharacterOffsetEnd=193 PartOfSpeech=JJ] [Text=sub CharacterOffsetBegin=194 CharacterOffsetEnd=197 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=198 CharacterOffsetEnd=200 PartOfSpeech=IN] [Text=his CharacterOffsetBegin=201 CharacterOffsetEnd=204 PartOfSpeech=PRP$] [Text=own CharacterOffsetBegin=205 CharacterOffsetEnd=208 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=208 CharacterOffsetEnd=209 PartOfSpeech=,] [Text=however CharacterOffsetBegin=210 CharacterOffsetEnd=217 PartOfSpeech=RB] [Text=; CharacterOffsetBegin=217 CharacterOffsetEnd=218 PartOfSpeech=:] [Text=he CharacterOffsetBegin=219 CharacterOffsetEnd=221 PartOfSpeech=PRP] [Text=has CharacterOffsetBegin=222 CharacterOffsetEnd=225 PartOfSpeech=VBZ] [Text=earn CharacterOffsetBegin=226 CharacterOffsetEnd=230 PartOfSpeech=VBP] [Text=it CharacterOffsetBegin=231 CharacterOffsetEnd=233 PartOfSpeech=PRP] [Text=by CharacterOffsetBegin=234 CharacterOffsetEnd=236 PartOfSpeech=IN] [Text=competing CharacterOffsetBegin=237 CharacterOffsetEnd=246 PartOfSpeech=VBG] [Text=against CharacterOffsetBegin=247 CharacterOffsetEnd=254 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=255 CharacterOffsetEnd=258 PartOfSpeech=DT] [Text=U.S. CharacterOffsetBegin=259 CharacterOffsetEnd=263 PartOfSpeech=NNP] [Text=Navy CharacterOffsetBegin=264 CharacterOffsetEnd=268 PartOfSpeech=NNP] [Text=in CharacterOffsetBegin=269 CharacterOffsetEnd=271 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=272 CharacterOffsetEnd=273 PartOfSpeech=DT] [Text=series CharacterOffsetBegin=274 CharacterOffsetEnd=280 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=281 CharacterOffsetEnd=283 PartOfSpeech=IN] [Text=war-games CharacterOffsetBegin=284 CharacterOffsetEnd=293 PartOfSpeech=NNS] [Text=with CharacterOffsetBegin=294 CharacterOffsetEnd=298 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=299 CharacterOffsetEnd=300 PartOfSpeech=DT] [Text=crew CharacterOffsetBegin=301 CharacterOffsetEnd=305 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=306 CharacterOffsetEnd=308 PartOfSpeech=IN] [Text=untested CharacterOffsetBegin=309 CharacterOffsetEnd=317 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=317 CharacterOffsetEnd=318 PartOfSpeech=,] [Text=questionable CharacterOffsetBegin=319 CharacterOffsetEnd=331 PartOfSpeech=JJ] [Text=, CharacterOffsetBegin=331 CharacterOffsetEnd=332 PartOfSpeech=,] [Text=and CharacterOffsetBegin=333 CharacterOffsetEnd=336 PartOfSpeech=CC] [Text=lovable CharacterOffsetBegin=337 CharacterOffsetEnd=344 PartOfSpeech=JJ] [Text=men CharacterOffsetBegin=345 CharacterOffsetEnd=348 PartOfSpeech=NNS] [Text=-LRB- CharacterOffsetBegin=349 CharacterOffsetEnd=350 PartOfSpeech=-LRB-] [Text=and CharacterOffsetBegin=350 CharacterOffsetEnd=353 PartOfSpeech=CC] [Text=a CharacterOffsetBegin=354 CharacterOffsetEnd=355 PartOfSpeech=DT] [Text=woman CharacterOffsetBegin=356 CharacterOffsetEnd=361 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=361 CharacterOffsetEnd=362 PartOfSpeech=,] [Text=too CharacterOffsetBegin=363 CharacterOffsetEnd=366 PartOfSpeech=RB] [Text=-RRB- CharacterOffsetBegin=366 CharacterOffsetEnd=367 PartOfSpeech=-RRB-] [Text=. CharacterOffsetBegin=367 CharacterOffsetEnd=368 PartOfSpeech=.] 
root(ROOT-0, adventure-5)
advmod(good-2, Surprisingly-1)
amod(adventure-5, good-2)
cc(good-2, and-3)
conj(good-2, quick-witted-4)
nsubj(features-7, that-6)
rcmod(adventure-5, features-7)
dobj(features-7, Kelsey-8)
appos(Kelsey-8, Grammer-9)
punct(supposedly-11, -LRB--10)
dep(Grammer-9, supposedly-11)
prep(supposedly-11, in-12)
poss(role-16, his-13)
amod(role-16, first-14)
amod(role-16, lead-15)
pobj(in-12, role-16)
prep(role-16, in-17)
det(feature-19, a-18)
pobj(in-17, feature-19)
punct(supposedly-11, -RRB--20)
prep(Grammer-9, as-21)
det(commander-26, an-22)
amod(commander-26, unconventional-23)
nn(commander-26, Navy-24)
nn(commander-26, sub-25)
pobj(as-21, commander-26)
nsubj(wants-28, who-27)
rcmod(commander-26, wants-28)
aux(run-30, to-29)
xcomp(wants-28, run-30)
det(sub-33, a-31)
amod(sub-33, nuclear-32)
dobj(run-30, sub-33)
prep(sub-33, of-34)
poss(own-36, his-35)
pobj(of-34, own-36)
punct(Kelsey-8, ,-37)
advmod(features-7, however-38)
punct(adventure-5, ;-39)
nsubj(has-41, he-40)
parataxis(adventure-5, has-41)
xcomp(has-41, earn-42)
dobj(earn-42, it-43)
prep(earn-42, by-44)
pcomp(by-44, competing-45)
prep(competing-45, against-46)
det(Navy-49, the-47)
nn(Navy-49, U.S.-48)
pobj(against-46, Navy-49)
prep(competing-45, in-50)
det(series-52, a-51)
pobj(in-50, series-52)
prep(series-52, of-53)
pobj(of-53, war-games-54)
prep(series-52, with-55)
det(crew-57, a-56)
pobj(with-55, crew-57)
prep(crew-57, of-58)
pobj(of-58, untested-59)
punct(untested-59, ,-60)
conj(untested-59, questionable-61)
punct(untested-59, ,-62)
cc(untested-59, and-63)
amod(men-65, lovable-64)
conj(untested-59, men-65)
punct(woman-69, -LRB--66)
cc(woman-69, and-67)
det(woman-69, a-68)
dep(men-65, woman-69)
punct(woman-69, ,-70)
advmod(woman-69, too-71)
punct(woman-69, -RRB--72)
punct(adventure-5, .-73)

Sentence #2 (38 tokens):
Director David S. Ward ("Major League") and his writers have put together a hysterically clever and upbeat comedy that tries to make the film easy to understand, which they do quite well.
[Text=Director CharacterOffsetBegin=368 CharacterOffsetEnd=376 PartOfSpeech=NNP] [Text=David CharacterOffsetBegin=377 CharacterOffsetEnd=382 PartOfSpeech=NNP] [Text=S. CharacterOffsetBegin=383 CharacterOffsetEnd=385 PartOfSpeech=NNP] [Text=Ward CharacterOffsetBegin=386 CharacterOffsetEnd=390 PartOfSpeech=NNP] [Text=-LRB- CharacterOffsetBegin=391 CharacterOffsetEnd=392 PartOfSpeech=-LRB-] [Text=`` CharacterOffsetBegin=392 CharacterOffsetEnd=393 PartOfSpeech=``] [Text=Major CharacterOffsetBegin=393 CharacterOffsetEnd=398 PartOfSpeech=NNP] [Text=League CharacterOffsetBegin=399 CharacterOffsetEnd=405 PartOfSpeech=NNP] [Text='' CharacterOffsetBegin=405 CharacterOffsetEnd=406 PartOfSpeech=''] [Text=-RRB- CharacterOffsetBegin=406 CharacterOffsetEnd=407 PartOfSpeech=-RRB-] [Text=and CharacterOffsetBegin=408 CharacterOffsetEnd=411 PartOfSpeech=CC] [Text=his CharacterOffsetBegin=412 CharacterOffsetEnd=415 PartOfSpeech=PRP$] [Text=writers CharacterOffsetBegin=416 CharacterOffsetEnd=423 PartOfSpeech=NNS] [Text=have CharacterOffsetBegin=424 CharacterOffsetEnd=428 PartOfSpeech=VBP] [Text=put CharacterOffsetBegin=429 CharacterOffsetEnd=432 PartOfSpeech=VBN] [Text=together CharacterOffsetBegin=433 CharacterOffsetEnd=441 PartOfSpeech=RP] [Text=a CharacterOffsetBegin=442 CharacterOffsetEnd=443 PartOfSpeech=DT] [Text=hysterically CharacterOffsetBegin=444 CharacterOffsetEnd=456 PartOfSpeech=RB] [Text=clever CharacterOffsetBegin=457 CharacterOffsetEnd=463 PartOfSpeech=JJ] [Text=and CharacterOffsetBegin=464 CharacterOffsetEnd=467 PartOfSpeech=CC] [Text=upbeat CharacterOffsetBegin=468 CharacterOffsetEnd=474 PartOfSpeech=JJ] [Text=comedy CharacterOffsetBegin=475 CharacterOffsetEnd=481 PartOfSpeech=NN] [Text=that CharacterOffsetBegin=482 CharacterOffsetEnd=486 PartOfSpeech=WDT] [Text=tries CharacterOffsetBegin=487 CharacterOffsetEnd=492 PartOfSpeech=VBZ] [Text=to CharacterOffsetBegin=493 CharacterOffsetEnd=495 PartOfSpeech=TO] [Text=make CharacterOffsetBegin=496 CharacterOffsetEnd=500 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=501 CharacterOffsetEnd=504 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=505 CharacterOffsetEnd=509 PartOfSpeech=NN] [Text=easy CharacterOffsetBegin=510 CharacterOffsetEnd=514 PartOfSpeech=JJ] [Text=to CharacterOffsetBegin=515 CharacterOffsetEnd=517 PartOfSpeech=TO] [Text=understand CharacterOffsetBegin=518 CharacterOffsetEnd=528 PartOfSpeech=VB] [Text=, CharacterOffsetBegin=528 CharacterOffsetEnd=529 PartOfSpeech=,] [Text=which CharacterOffsetBegin=530 CharacterOffsetEnd=535 PartOfSpeech=WDT] [Text=they CharacterOffsetBegin=536 CharacterOffsetEnd=540 PartOfSpeech=PRP] [Text=do CharacterOffsetBegin=541 CharacterOffsetEnd=543 PartOfSpeech=VBP] [Text=quite CharacterOffsetBegin=544 CharacterOffsetEnd=549 PartOfSpeech=RB] [Text=well CharacterOffsetBegin=550 CharacterOffsetEnd=554 PartOfSpeech=RB] [Text=. CharacterOffsetBegin=554 CharacterOffsetEnd=555 PartOfSpeech=.] 
root(ROOT-0, put-15)
nn(Ward-4, Director-1)
nn(Ward-4, David-2)
nn(Ward-4, S.-3)
nsubj(put-15, Ward-4)
punct(League-8, -LRB--5)
punct(League-8, ``-6)
nn(League-8, Major-7)
dep(Ward-4, League-8)
punct(League-8, ''-9)
punct(League-8, -RRB--10)
cc(Ward-4, and-11)
poss(writers-13, his-12)
conj(Ward-4, writers-13)
aux(put-15, have-14)
prt(put-15, together-16)
det(comedy-22, a-17)
advmod(clever-19, hysterically-18)
amod(comedy-22, clever-19)
cc(clever-19, and-20)
conj(clever-19, upbeat-21)
dobj(put-15, comedy-22)
nsubj(tries-24, that-23)
rcmod(comedy-22, tries-24)
aux(make-26, to-25)
xcomp(tries-24, make-26)
det(film-28, the-27)
nsubj(easy-29, film-28)
xcomp(make-26, easy-29)
aux(understand-31, to-30)
xcomp(easy-29, understand-31)
punct(understand-31, ,-32)
dobj(do-35, which-33)
nsubj(do-35, they-34)
ccomp(understand-31, do-35)
advmod(well-37, quite-36)
advmod(do-35, well-37)
punct(put-15, .-38)

Sentence #3 (29 tokens):
Second, besides the Grammer character, most of the crew provide some or plenty of the movie's humor, which is treated like a double-edged sword.
[Text=Second CharacterOffsetBegin=560 CharacterOffsetEnd=566 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=566 CharacterOffsetEnd=567 PartOfSpeech=,] [Text=besides CharacterOffsetBegin=568 CharacterOffsetEnd=575 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=576 CharacterOffsetEnd=579 PartOfSpeech=DT] [Text=Grammer CharacterOffsetBegin=580 CharacterOffsetEnd=587 PartOfSpeech=NNP] [Text=character CharacterOffsetBegin=588 CharacterOffsetEnd=597 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=597 CharacterOffsetEnd=598 PartOfSpeech=,] [Text=most CharacterOffsetBegin=599 CharacterOffsetEnd=603 PartOfSpeech=JJS] [Text=of CharacterOffsetBegin=604 CharacterOffsetEnd=606 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=607 CharacterOffsetEnd=610 PartOfSpeech=DT] [Text=crew CharacterOffsetBegin=611 CharacterOffsetEnd=615 PartOfSpeech=NN] [Text=provide CharacterOffsetBegin=616 CharacterOffsetEnd=623 PartOfSpeech=VBP] [Text=some CharacterOffsetBegin=624 CharacterOffsetEnd=628 PartOfSpeech=DT] [Text=or CharacterOffsetBegin=629 CharacterOffsetEnd=631 PartOfSpeech=CC] [Text=plenty CharacterOffsetBegin=632 CharacterOffsetEnd=638 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=639 CharacterOffsetEnd=641 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=642 CharacterOffsetEnd=645 PartOfSpeech=DT] [Text=movie CharacterOffsetBegin=646 CharacterOffsetEnd=651 PartOfSpeech=NN] [Text='s CharacterOffsetBegin=651 CharacterOffsetEnd=653 PartOfSpeech=POS] [Text=humor CharacterOffsetBegin=654 CharacterOffsetEnd=659 PartOfSpeech=NN] [Text=, CharacterOffsetBegin=659 CharacterOffsetEnd=660 PartOfSpeech=,] [Text=which CharacterOffsetBegin=661 CharacterOffsetEnd=666 PartOfSpeech=WDT] [Text=is CharacterOffsetBegin=667 CharacterOffsetEnd=669 PartOfSpeech=VBZ] [Text=treated CharacterOffsetBegin=670 CharacterOffsetEnd=677 PartOfSpeech=VBN] [Text=like CharacterOffsetBegin=678 CharacterOffsetEnd=682 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=683 CharacterOffsetEnd=684 PartOfSpeech=DT] [Text=double-edged CharacterOffsetBegin=685 CharacterOffsetEnd=697 PartOfSpeech=JJ] [Text=sword CharacterOffsetBegin=698 CharacterOffsetEnd=703 PartOfSpeech=NN] [Text=. CharacterOffsetBegin=703 CharacterOffsetEnd=704 PartOfSpeech=.] 
root(ROOT-0, Second-1)
punct(Second-1, ,-2)
dep(,-2, besides-3)
det(character-6, the-4)
nn(character-6, Grammer-5)
pobj(besides-3, character-6)
punct(character-6, ,-7)
nsubj(provide-12, most-8)
prep(most-8, of-9)
det(crew-11, the-10)
pobj(of-9, crew-11)
rcmod(character-6, provide-12)
dobj(provide-12, some-13)
cc(some-13, or-14)
conj(some-13, plenty-15)
prep(some-13, of-16)
det(movie-18, the-17)
poss(humor-20, movie-18)
possessive(movie-18, 's-19)
pobj(of-16, humor-20)
punct(some-13, ,-21)
nsubjpass(treated-24, which-22)
auxpass(treated-24, is-23)
rcmod(some-13, treated-24)
prep(treated-24, like-25)
det(sword-28, a-26)
amod(sword-28, double-edged-27)
pobj(like-25, sword-28)
punct(besides-3, .-29)

Sentence #4 (35 tokens):
Three members of the supporting cast - Harland Williams, Toby Huss, and Rob Schneider provide the funniest scenes, which includes weird gestures, impersonations, and, well, their personalities.
[Text=Three CharacterOffsetBegin=709 CharacterOffsetEnd=714 PartOfSpeech=CD] [Text=members CharacterOffsetBegin=715 CharacterOffsetEnd=722 PartOfSpeech=NNS] [Text=of CharacterOffsetBegin=723 CharacterOffsetEnd=725 PartOfSpeech=IN] [Text=the CharacterOffsetBegin=726 CharacterOffsetEnd=729 PartOfSpeech=DT] [Text=supporting CharacterOffsetBegin=730 CharacterOffsetEnd=740 PartOfSpeech=VBG] [Text=cast CharacterOffsetBegin=741 CharacterOffsetEnd=745 PartOfSpeech=NN] [Text=- CharacterOffsetBegin=746 CharacterOffsetEnd=747 PartOfSpeech=:] [Text=Harland CharacterOffsetBegin=748 CharacterOffsetEnd=755 PartOfSpeech=NNP] [Text=Williams CharacterOffsetBegin=756 CharacterOffsetEnd=764 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=764 CharacterOffsetEnd=765 PartOfSpeech=,] [Text=Toby CharacterOffsetBegin=766 CharacterOffsetEnd=770 PartOfSpeech=NNP] [Text=Huss CharacterOffsetBegin=771 CharacterOffsetEnd=775 PartOfSpeech=NNP] [Text=, CharacterOffsetBegin=775 CharacterOffsetEnd=776 PartOfSpeech=,] [Text=and CharacterOffsetBegin=777 CharacterOffsetEnd=780 PartOfSpeech=CC] [Text=Rob CharacterOffsetBegin=781 CharacterOffsetEnd=784 PartOfSpeech=NNP] [Text=Schneider CharacterOffsetBegin=785 CharacterOffsetEnd=794 PartOfSpeech=NNP] [Text=provide CharacterOffsetBegin=795 CharacterOffsetEnd=802 PartOfSpeech=VBP] [Text=the CharacterOffsetBegin=803 CharacterOffsetEnd=806 PartOfSpeech=DT] [Text=funniest CharacterOffsetBegin=807 CharacterOffsetEnd=815 PartOfSpeech=JJS] [Text=scenes CharacterOffsetBegin=816 CharacterOffsetEnd=822 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=822 CharacterOffsetEnd=823 PartOfSpeech=,] [Text=which CharacterOffsetBegin=824 CharacterOffsetEnd=829 PartOfSpeech=WDT] [Text=includes CharacterOffsetBegin=830 CharacterOffsetEnd=838 PartOfSpeech=VBZ] [Text=weird CharacterOffsetBegin=839 CharacterOffsetEnd=844 PartOfSpeech=JJ] [Text=gestures CharacterOffsetBegin=845 CharacterOffsetEnd=853 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=853 CharacterOffsetEnd=854 PartOfSpeech=,] [Text=impersonations CharacterOffsetBegin=855 CharacterOffsetEnd=869 PartOfSpeech=NNS] [Text=, CharacterOffsetBegin=869 CharacterOffsetEnd=870 PartOfSpeech=,] [Text=and CharacterOffsetBegin=871 CharacterOffsetEnd=874 PartOfSpeech=CC] [Text=, CharacterOffsetBegin=874 CharacterOffsetEnd=875 PartOfSpeech=,] [Text=well CharacterOffsetBegin=876 CharacterOffsetEnd=880 PartOfSpeech=RB] [Text=, CharacterOffsetBegin=880 CharacterOffsetEnd=881 PartOfSpeech=,] [Text=their CharacterOffsetBegin=882 CharacterOffsetEnd=887 PartOfSpeech=PRP$] [Text=personalities CharacterOffsetBegin=888 CharacterOffsetEnd=901 PartOfSpeech=NNS] [Text=. CharacterOffsetBegin=901 CharacterOffsetEnd=902 PartOfSpeech=.] 
root(ROOT-0, members-2)
num(members-2, Three-1)
prep(members-2, of-3)
pobj(of-3, the-4)
partmod(the-4, supporting-5)
nsubj(provide-17, cast-6)
punct(cast-6, --7)
nn(Williams-9, Harland-8)
dep(cast-6, Williams-9)
punct(Williams-9, ,-10)
nn(Huss-12, Toby-11)
appos(Williams-9, Huss-12)
punct(Williams-9, ,-13)
cc(Williams-9, and-14)
nn(Schneider-16, Rob-15)
conj(Williams-9, Schneider-16)
ccomp(supporting-5, provide-17)
det(scenes-20, the-18)
amod(scenes-20, funniest-19)
dobj(provide-17, scenes-20)
punct(scenes-20, ,-21)
nsubj(includes-23, which-22)
rcmod(scenes-20, includes-23)
amod(gestures-25, weird-24)
dobj(includes-23, gestures-25)
punct(gestures-25, ,-26)
conj(gestures-25, impersonations-27)
punct(gestures-25, ,-28)
cc(gestures-25, and-29)
punct(gestures-25, ,-30)
advmod(personalities-34, well-31)
punct(personalities-34, ,-32)
poss(personalities-34, their-33)
conj(gestures-25, personalities-34)
punct(members-2, .-35)

Sentence #5 (22 tokens):
"Down Periscope" is more than a Navy version of "Police Academy" or "The Dirty Dozen".
[Text=`` CharacterOffsetBegin=902 CharacterOffsetEnd=903 PartOfSpeech=``] [Text=Down CharacterOffsetBegin=903 CharacterOffsetEnd=907 PartOfSpeech=NN] [Text=Periscope CharacterOffsetBegin=908 CharacterOffsetEnd=917 PartOfSpeech=NN] [Text='' CharacterOffsetBegin=917 CharacterOffsetEnd=918 PartOfSpeech=''] [Text=is CharacterOffsetBegin=919 CharacterOffsetEnd=921 PartOfSpeech=VBZ] [Text=more CharacterOffsetBegin=922 CharacterOffsetEnd=926 PartOfSpeech=JJR] [Text=than CharacterOffsetBegin=927 CharacterOffsetEnd=931 PartOfSpeech=IN] [Text=a CharacterOffsetBegin=932 CharacterOffsetEnd=933 PartOfSpeech=DT] [Text=Navy CharacterOffsetBegin=934 CharacterOffsetEnd=938 PartOfSpeech=NNP] [Text=version CharacterOffsetBegin=939 CharacterOffsetEnd=946 PartOfSpeech=NN] [Text=of CharacterOffsetBegin=947 CharacterOffsetEnd=949 PartOfSpeech=IN] [Text=`` CharacterOffsetBegin=950 CharacterOffsetEnd=951 PartOfSpeech=``] [Text=Police CharacterOffsetBegin=951 CharacterOffsetEnd=957 PartOfSpeech=NNP] [Text=Academy CharacterOffsetBegin=958 CharacterOffsetEnd=965 PartOfSpeech=NNP] [Text='' CharacterOffsetBegin=965 CharacterOffsetEnd=966 PartOfSpeech=''] [Text=or CharacterOffsetBegin=967 CharacterOffsetEnd=969 PartOfSpeech=CC] [Text=`` CharacterOffsetBegin=970 CharacterOffsetEnd=971 PartOfSpeech=``] [Text=The CharacterOffsetBegin=971 CharacterOffsetEnd=974 PartOfSpeech=DT] [Text=Dirty CharacterOffsetBegin=975 CharacterOffsetEnd=980 PartOfSpeech=NNP] [Text=Dozen CharacterOffsetBegin=981 CharacterOffsetEnd=986 PartOfSpeech=NNP] [Text='' CharacterOffsetBegin=986 CharacterOffsetEnd=987 PartOfSpeech=''] [Text=. CharacterOffsetBegin=987 CharacterOffsetEnd=988 PartOfSpeech=.] 
root(ROOT-0, more-6)
punct(Periscope-3, ``-1)
nn(Periscope-3, Down-2)
nsubj(more-6, Periscope-3)
punct(Periscope-3, ''-4)
cop(more-6, is-5)
prep(more-6, than-7)
det(version-10, a-8)
nn(version-10, Navy-9)
pobj(than-7, version-10)
prep(version-10, of-11)
punct(Academy-14, ``-12)
nn(Academy-14, Police-13)
pobj(of-11, Academy-14)
punct(Academy-14, ''-15)
cc(Academy-14, or-16)
punct(Academy-14, ``-17)
det(Dozen-20, The-18)
nn(Dozen-20, Dirty-19)
conj(Academy-14, Dozen-20)
punct(Academy-14, ''-21)
punct(more-6, .-22)

Sentence #6 (17 tokens):
What the film does is get the job done the right way and I liked it.
[Text=What CharacterOffsetBegin=993 CharacterOffsetEnd=997 PartOfSpeech=WP] [Text=the CharacterOffsetBegin=998 CharacterOffsetEnd=1001 PartOfSpeech=DT] [Text=film CharacterOffsetBegin=1002 CharacterOffsetEnd=1006 PartOfSpeech=NN] [Text=does CharacterOffsetBegin=1007 CharacterOffsetEnd=1011 PartOfSpeech=VBZ] [Text=is CharacterOffsetBegin=1012 CharacterOffsetEnd=1014 PartOfSpeech=VBZ] [Text=get CharacterOffsetBegin=1015 CharacterOffsetEnd=1018 PartOfSpeech=VB] [Text=the CharacterOffsetBegin=1019 CharacterOffsetEnd=1022 PartOfSpeech=DT] [Text=job CharacterOffsetBegin=1023 CharacterOffsetEnd=1026 PartOfSpeech=NN] [Text=done CharacterOffsetBegin=1027 CharacterOffsetEnd=1031 PartOfSpeech=VBN] [Text=the CharacterOffsetBegin=1032 CharacterOffsetEnd=1035 PartOfSpeech=DT] [Text=right CharacterOffsetBegin=1036 CharacterOffsetEnd=1041 PartOfSpeech=JJ] [Text=way CharacterOffsetBegin=1042 CharacterOffsetEnd=1045 PartOfSpeech=NN] [Text=and CharacterOffsetBegin=1046 CharacterOffsetEnd=1049 PartOfSpeech=CC] [Text=I CharacterOffsetBegin=1050 CharacterOffsetEnd=1051 PartOfSpeech=PRP] [Text=liked CharacterOffsetBegin=1052 CharacterOffsetEnd=1057 PartOfSpeech=VBD] [Text=it CharacterOffsetBegin=1058 CharacterOffsetEnd=1060 PartOfSpeech=PRP] [Text=. CharacterOffsetBegin=1060 CharacterOffsetEnd=1061 PartOfSpeech=.] 
root(ROOT-0, get-6)
dobj(does-4, What-1)
det(film-3, the-2)
nsubj(does-4, film-3)
csubj(get-6, does-4)
aux(get-6, is-5)
det(job-8, the-7)
nsubj(done-9, job-8)
dep(get-6, done-9)
det(way-12, the-10)
amod(way-12, right-11)
dobj(done-9, way-12)
cc(done-9, and-13)
nsubj(liked-15, I-14)
conj(done-9, liked-15)
dobj(liked-15, it-16)
punct(get-6, .-17)

